docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,253 +8,318 @@ weight: 200
+1. Create a `docker-compose.yaml` file 
+   
+1. Add the following code to the into it:
+   ```yaml
+   version: "3.8"
+    services:
+      grafana:
+        image: grafana/grafana-enterprise
+        container_name: grafana
+        restart: unless-stopped
+        ports:
+          - '3000:3000'
+    ```
 
-**Warning**: When migrating from an earlier version to 5.1 or later using Docker compose and implicit volumes, you need to use `docker inspect` to find out which volumes your container is mapped to so that you can map them to the upgraded container as well. You will also have to change file ownership (or user) as documented below.
+1. Save the file and run the following command:
 
-#### User ID changes
+   ```bash
+   docker compose up -d docker-compose.yaml
+    ```
 
-In Grafana v5.1, we changed the ID and group of the Grafana user and in v7.3 we changed the group. Unfortunately this means that files created prior to v5.1 won't have the correct permissions for later versions. We made this change so that it would be more likely that the Grafana users ID would be unique to Grafana. For example, on Ubuntu 16.04 `104` is already in use by the syslog user.
+#### Use bind mounts
 
-| Version | User    | User ID | Group   | Group ID |
-| ------- | ------- | ------- | ------- | -------- |
-| < 5.1   | grafana | 104     | grafana | 107      |
-| \>= 5.1 | grafana | 472     | grafana | 472      |
-| \>= 7.3 | grafana | 472     | root    | 0        |
+If you plan to use folders on your host for the database or configuration when running Grafana in Docker, you must start the container with a user that has the permission to access and write to the folder you map.
 
-There are two possible solutions to this problem. Either you start the new container as the root user and change ownership from `104` to `472`, or you start the upgraded container as user `104`.
+To use bind mounts, complete the following steps:
 
-##### Run Docker as a different user
+1. Create a new `docker-compose.yaml` file
 
-```bash
-docker run --user 104 --volume "<your volume mapping here>" grafana/grafana-enterprise:8.2.0
-```
+1. Add the following code into it:
 
-##### Specify a user in docker-compose.yml
+   ```yaml
+   services:
+     grafana:
+       image: grafana/grafana-enterprise
+       container_name: grafana
+       restart: unless-stopped
+       # if you are running as root then set it to 0
+       # else find the right id with the id -u command
+       user: "0"
+       ports:
+         - '3000:3000'
+       volumes:
+         - '$PWD/data:/var/lib/grafana'
 
-```yaml
-version: '2'
+1. Save the file and run the following command:
 
-services:
-  grafana:
-    image: grafana/grafana-enterprise:8.2.0
-    ports:
-      - 3000:3000
-    user: '104'
-```
+   ```bash
+   docker compose up -d docker-compose.yaml
+    ```
 
```

Comment:
@usmangt , I notice that there is no Install plugins in the Docker container section here within Docker Compose. Should there be?

Reply:
Yes, there aren't as wanted to explain to users how to use different storage strategies e.g. persistent or bind mount as simple as possible. Just below this section, is the Example heading which contains how to install a plugin.